### ISSUE TITLE
Re-add animated revive icon

### ISSUE THREAD

**Issue by SamLex** [Raptoer/F3#110]
After #109, the animated revive icon that indicates closeness to death and revive status will no longer be shown to the player. This should be re-added in a non-confusing manner.

**Comment by SamLex** [comment]
I believe a Revive animation is being added in Tanks DLC, so this maybe unneeded.

**Comment by SamLex** [comment]
With the addition of the Revive animation in Tanks, I feel this is now unneeded or, at the very least, very low priority. So to the backlog with thee

**Comment by SamLex** [comment]
Closing as I don't think this is needed anymore
